packages/react-components/react-toast/stories/Toast/UpdateToast.stories.tsx

```diff
@@ -0,0 +1,16 @@
+import * as React from 'react';
+import { Toaster, useToastController, ToastId } from '@fluentui/react-toast';
+
+export const UpdateToast = () => {
+  const { dispatchToast, updateToast } = useToastController();
+  const notify = (id: ToastId) => dispatchToast('This toast never closes', { toastId: id, timeout: -1 });
+  const update = (id: ToastId) => updateToast({ content: 'This toast will close soon', toastId: id, timeout: 1000 });
+
+  return (
+    <>
+      <Toaster />
+      <button onClick={() => notify('example')}>Make toast</button>
+      <button onClick={() => update('example')}>Update toast</button>
```

Comment:
```suggestion
      <button onClick={() => notify('EXAMPLE_ID')}>Make toast</button>
      <button onClick={() => update('EXAMPLE_ID')}>Update toast</button>
```

Let's be more explicit about IDs there 🐱

Reply:
You can also use `useId()` hook to generate id